### DISCUSSION TITLE
[求肋]绳网头像怎么换啊

### DISCUSSION THREAD

**Discussion by ijog36** [share121/inter-knot#1223] [其他]
做为从仙舟来的人对于另一种语言感受到了巨大的挑战
![Uploading Image_1722054510422.jpg…]()

**Comment by SaKura-071721** [comment]
从github上改就好了

**Comment by lllliufeng** [comment]
点你绳网上的头像跳转到github，再点头像跳转到编辑主页，点edit，upload a photo
_我是一个复读机_
